app/assets/javascripts/cable.js

```diff
@@ -0,0 +1,13 @@
+// Action Cable provides the framework to deal with WebSockets in Rails.
```

Comment:
@tpitale 99% sure we won't. Given that, probably worth ditching:

- `app/assets/javascripts/channels`
- `app/assets/javascripts/cable.js`
- `app/channels`

…yeah?